cukinia

```diff
@@ -229,6 +230,17 @@ _cukinia_http_request()
 	wget -q -O /dev/null "$url"
 }
 
+# _cukinia_symlink: validate the target of symlink
+# args1: link
+# args2: target
+_cukinia_symlink()
+{
+	local link="$1"
+	local target="$2"
+	_cukinia_prepare "Checking for $link pointing to $target"
+	[ $(readlink "$link") == "$target" ] && return 0 || return 1
```

Comment:
* use = to compare strings in posix sh - see dash(1) "s1 = s2"
* ensure both sides are double-quoted in case one string is empty (`readlink $link`) may return <empty>